modules/core/src/main/scala/hotpotato/ErrorTrans.scala

```diff
@@ -23,11 +21,13 @@ trait ErrorTransThrow[F[_, _]] extends ErrorTrans[F] {
   ): F[LL, R]
 }
 
-object ErrorTrans extends ErrorTransInstances {}
+object ErrorTrans extends ErrorTransInstances {
+  def apply[F[_, _]: ErrorTrans]: ErrorTrans[F] = implicitly
+}
 
-private[hotpotato] trait ErrorTransInstances extends ErrorTransLowerInstances {
+private[hotpotato] trait ErrorTransInstances {
 
-  implicit val eitherErrorTrans: ErrorTrans[Either] =
+  implicit val either: ErrorTrans[Either] =
```

Comment:
Can you keep the name as they were? It allows users to specify the instance explicitly with less ambiguity

Reply:
Okay but what is ambiguous about `ErrorTrans.either`?

Reply:
Now your namespace has a random `either` variable :sweat_smile: